### PR TITLE
Update azure_rm_functionapp related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_functionapp related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_functionapp/azure_rm_functionapp_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_functionapp.py [hunk 1]
```diff
@@ -38,8 +38,8 @@
         description:
             - App service plan.
             - It can be name of existing app service plan in same resource group as function app.
-            - "It can be resource id of existing app service plan. eg.,
-              /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>"
+            - It can be resource id of existing app service plan.
+            - Reource id, eg. C(/subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>).
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_functionapp.py]: should this enclosed with C()?

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_functionapp.py]: It's my mistake, the “C()” should used to files and option value, I will changed this!  Thank you very much!

File: lib/ansible/modules/cloud/azure/azure_rm_functionapp.py [hunk 2]
```diff
@@ -38,8 +38,8 @@
         description:
             - App service plan.
             - It can be name of existing app service plan in same resource group as function app.
-            - "It can be resource id of existing app service plan. eg.,
-              /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>"
+            - It can be resource id of existing app service plan.
+            - Reource id, eg. /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>)
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_functionapp.py]:
```suggestion
            - Resource id, eg. /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>)
```

**Review by acozine — COMMENTED** [review]
Typo.

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Thanks @Fred-sun!

**Pull request merged**